src/tools/sdk/archive/index.md

```diff
@@ -1,13 +1,14 @@
 ---
-title: "Index of Downloads"
-description: "Download specific stable and dev channel versions of the Dart SDK, Dartium, and the Dart API documentation."
-permalink: /install/archive
+title: Dart SDK Download
+short-title: Download
+description: Download specific stable and dev channel versions of the Dart SDK and the Dart API documentation.
 js:
-- url: /install/archive/out/web/download_archive.dart.js
+- url: /tools/sdk/archive/out/web/download_archive.dart.js
   defer: true
-- url: /install/archive/assets/install.js
+- url: /tools/sdk/archive/assets/install.js
   defer: true
 ---
+
```

Comment:
Later this page refers to the "main Dart installation page". Let's just call it the "Dart SDK" page. We should also check for other references to that page.

Reply:
> We should also check for other references to that page.

I've swept the docs and made the necessary changes.